content/03.methods.md

```diff
@@ -5,8 +5,59 @@ Talk about dataset - Pubtator
 Talk about preprocessing Pubtator
 Talk about hand annotations for each realtion
 
-## Label Functions
-describe what a label function is and how many we created for each relation
+## Label Functions for Annotating Sentences
+We used simple pythonic functions called label functions to quickly annotate our candidates sentences.
+These functions are designed to emit: a positive label (1), a negative label (-1) 
+or abstain from emitting a label (0).
+We constructed label functions in the following three categories (Databases, Text Patterns and Domain Heuristics).
+These categories are described below using the following candidate sentence as an example: \"<span style="color:blue">PTK6</span> may be a novel therapeutic target for <span style="color:brown">pancreatic cancer</span>."
+
+**Databases**: This category of label functions are designed to utilize publicly available databases to determine if a co-mention pair has been previously established.
+It emits a positive label if the co-mention pair is located within a database, otherwise the given label function will abstain.
+If the label function abstains, another label function will emit a negative label.
```

Comment:
I don't quite get what this is going at without the context of our past conversations - re-word